modules/vault-elb/variables.tf

```diff
@@ -25,6 +25,20 @@ variable "vault_asg_name" {
 # These parameters have reasonable defaults.
 # ---------------------------------------------------------------------------------------------------------------------
 
+# Due to terraform type limitations this variable is set to be a map of any.
+# In reality it is a map of mixed parameters, some of them are booleans, some ints.
+#  access_logs = {
+#    enabled = true # mandatory
+#    bucket = "s3_bucket_name" # mandatory
+#    bucket_prefix = "alb" # mandatory
+#    interval = 60 # optional
```

Comment:
NIT: `fmt` this example for easier reading.

```suggestion
#    enabled = true # mandatory
#    bucket = "s3_bucket_name" # mandatory
#    bucket_prefix = "alb" # mandatory
#    interval = 60 # optional
```

Reply:
Whoops, I didn't mean to include the suggested formatting in there! But, as it's there, I've now updated to actually look as if it's had `fmt` run on it to make it easier to read.